docs/webdev-setup.md

```
@@ -0,0 +1,423 @@
+        data_type = data.get('dataType')
+        if data_type:
+            value = system.tag.DataType.valueOf(data_type).coerce(value)
+        system.tag.writeBlocking([path], [value])
+        return {'json': {'status': 'ok', 'tagPath': path, 'value': value}}
+
+    else:
+        return {'json': {'error': 'Invalid request body'}, 'response': {'code': 400}}
+```
+
+**Configuration:**
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)